ihatemoney/templates/statistics.html

```diff
@@ -33,6 +33,243 @@ <h2>{{ _("Expenses by Month") }}</h2>
         {% endfor %}
         </tbody>
         </table>
+        <h2> {{ _("Visualization") }}</h2>
+        
+        
+        <!-- Required for some style in charts, feel free to move from here -->
+        <style>
+        #balanceBar .bb-ygrid-line line {stroke: red;}
+
+        #balanceBar .fill_green { fill: green; }
+        #balanceBar .fill_red { fill: red; }
+        </style>
+        <br>
+        <div id="balanceBar"></div>
```

Comment:
As I said in a comment, I'm for removing this one as it doesn't add much information.